README.md

```diff
@@ -72,6 +72,70 @@ This action requires the following minimum set of permissions:
 
 Note: the policy above assumes the account has opted in to the ECS long ARN format.
 
+## CodeDeploy Support
```

Comment:
nit: should be "AWS CodeDeploy"? (here and elsewhere in this doc)

Reply:
ah ha, yes

Reply:
Done